types/lhr/settings.d.ts

```diff
@@ -73,6 +73,8 @@ export type ScreenEmulationSettings = {
   debugNavigation?: boolean;
   /** If set to true, will skip the initial navigation to about:blank. This option is ignored when using the legacy navigation runner. */
   skipAboutBlank?: boolean;
+  /** If set to true, gatherers should avoid any behavior that may be destructive to the page state. (e.g. extra navigations, resizing the viewport) */
+  usePassiveGathering?: boolean;
```

Comment:
should inclusion in this type imply it should be available via the cli?

Reply:
We never added `skipAboutBlank` because it isn't useful (cli starts on about:blank anyway).

For similar reasons `usePassiveGathering` isn't useful on the cli because it will always test a standalone navigation and then close the page. Preserving the page state isn't useful in this scenario.

That being said, it wouldn't be difficult to include just for consistency with other options on this type.